resources/tools/INSTALL.txt

```diff
@@ -0,0 +1,46 @@
+                                                    ~ BALLERINA ~
+                                                    =============
+
+
+PREREQUISITES
+-------------
+
+Ballerina requires JRE 8 Update 172+.
+It is recommended to use AdoptOpenJDK. Check https://adoptopenjdk.net/releases.html to download the JRE.
```

Comment:
Shall we say download jre from here? https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/tag/jdk8u202-b08